lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -44,8 +44,7 @@
                        this.$('a.is-active').removeClass('is-active').attr('aria-selected', 'false');
                        this.$('a[data-index='+index+']').addClass('is-active').attr('aria-selected', 'true');
                        var view = this.tabs[index].view;
-                       view.render();
-                       this.$('.page-content-main').html(view.$el.html());
+                       view.setElement(this.$('.page-content-main')).render();
```

Comment:
@dan-f That div comes from [`lms/templates/components/tabbed/tabbed_view.underscore`](https://github.com/edx/edx-platform/blob/master/lms/templates/components/tabbed/tabbed_view.underscore#L4), so, yes, it should.